src/sys/epoll.rs

```diff
@@ -26,6 +26,64 @@ libc_bitflags!(
     }
 );
 
+/// A safe wrapper around [`epoll`](https://man7.org/linux/man-pages/man7/epoll.7.html).
+/// ```
+/// # use nix::sys::{epoll::{Epoll, EpollEvent, EpollFlags, EpollCreateFlags}, eventfd::{eventfd, EfdFlags}};
+/// # use nix::unistd::write;
+/// # use std::os::unix::io::{OwnedFd, FromRawFd, AsRawFd, AsFd};
+/// # use std::time::{Instant, Duration};
+/// # fn main() -> nix::Result<()> {
```

Comment:
Do you need to declare the `main()` function explicitly?  If you leave it out, I think rustdoc will add it automatically.

Reply:
Rustdoc automatically adds `main() {}` without a return type. To use `?` requires manually writing it with a return type.